src/components/FileUpload/sgds-fileupload.ts

```diff
@@ -24,52 +24,69 @@ export type FileUploadButtonVariant =
   | "outline-light"
   | "outline-dark";
 
+/**
+ * @summary Allows users to upload files of various sizes and formats
+ * @slot default - Label for fileuploader button
+ *
+ * @event sgds-files-selected - Emitted when files are selected for uploading
+ *
+ * @cssproperty --fileuploader-file-icon-fill - Left icon fill color
+ * @cssproperty --fileuploader-remove-icon-fill - Remove icon fill color
+ * @cssproperty --fileuploader-remove-icon-hover-fill - Remove icon mouse over fill color
+ *
+ */
+
 @customElement("sgds-fileupload")
 export class SgdsFileUpload extends SgdsElement {
+  static styles = [SgdsElement.styles, styles];
+
   /** The button's variant. */
   @property({ reflect: true }) variant: FileUploadButtonVariant = "primary";
 
   // /** Sets a unique id to the file input, required. */
   // @property({ type: String }) controlId = "";
+  //** Disable the fileuploader button */
+  @property({ type: Boolean, reflect: true })
+  disabled: boolean = false;
 
+  /** Allows multiple files to be listed for uploading */
   @property({ type: Boolean, reflect: true })
-  disabled = false;
+  multiple: boolean = false;
 
-  /** Button sizes */
+  /** Specify the acceptable file type  */
+  @property({ type: String, reflect: true })
+  accept: string = "";
+
+  /** Specifies a large or small button */
   @property({ reflect: true }) size: "sm" | "lg";
 
+  /** Customize the check icon with SVG */
   @property({ type: String })
-  checkedIcon? = "";
+  checkedIcon: string = "";
 
+  /** Customize the cancel icon with SVG */
   @property({ type: String })
-  cancelIcon? = "";
+  cancelIcon: string = "";
 
+  /** @internal */
   @property({ type: FileList })
```

Comment:
Lit doesnt have a converter for type FileList.. since FileList is considered an Object, use `@property( {type: Object})` instead.  Since its private and more like a state, can use state option true. See https://lit.dev/docs/api/ReactiveElement/#PropertyDeclaration.state
This is only for line 72. Line 73 typescript typings are ok 

so change to : 
`  @property({ type: Object, state: true })`